requirements.txt

```diff
@@ -0,0 +1 @@
+ipython
```

Comment:
Why add a requirements file for this?
It is already included in the install_requires argument for setup() in setup.py.